moped-editor/src/queries/project.js

```diff
@@ -114,6 +114,13 @@ export const SUMMARY_QUERY = gql`
       status_id
       status_name
     }
+    moped_user_followed_projects(
+      where: { project_id: { _eq: $projectId }, user_id: { _eq: $userId} } 
+    ) {
+      project_id
+      user_id
+      id
```

Comment:
Would you consider not requesting back this ID from `graphql-engine`? It doesn't look like it gets used. I think not using it is correct -- that is to say, the way you delete a "following" record is perfect.